starwars/src/App.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import './App.css';
+import StarWarsData from './components/StarWarsData';
```

Comment:
StarWarsData is the final functional component according to your file structure, you should be importing your StarWarsList